src/discourse.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Interface for interactions with discourse."""
+import logging
+import typing
+
+from ops.pebble import ChangeError, ExecError, ExecProcess
+
+CONTAINER_APP_USERNAME = "_daemon_"
+DISCOURSE_PATH = "/srv/discourse/app"
```

Comment:
Same as above, we define this in `src/charm.py` currently.

Reply:
I replied to the above comment.